test/source/mock/key-manager/key-manager-endpoints.ts

```diff
@@ -218,8 +228,8 @@ export const mockKeyManagerEndpoints: HandlersDefinition = {
       if (acctEmail === 'get.key@key-manager-autogen.flowcrypt.test') {
         return { privateKeys: [{ decryptedPrivateKey: testConstants.existingPrv }] };
       }
-      if (acctEmail.includes('updating.key')) {
-        const { response, badRequestError } = MOCK_KM_UPDATING_KEY[acctEmail];
+      if (acctEmail.includes('updating.key') || acctEmail === 'flowcrypt.notify.expiring.keys@key-manager-autogen.flowcrypt.test') {
```

Comment:
Couldn't we just use the marker `updating.key` (or any other) for the tests that use `MOCK_KM_KEYS`?